lib/pry-doc/pry_ext/method_info.rb

```diff
@@ -186,6 +186,12 @@ def cache(meth)
           YARD.parse(file)
         end
       end
+
+      private
+
+      def namespace_name(host)
```

Comment:
Module#namespace_name is  removed https://github.com/lsegal/yard/pull/1229. So, I implement pry-doc as private method.